web/src/components/SourceFilesView/sourceFilesView.js

```diff
@@ -52,13 +52,21 @@ class SourceFilesView extends Component {
         }
       }).then(response => {
         const sourceFiles = response.data.reduce( (files, file) => {
-          const fileName = file.filename.split("/").splice(-1)[0];
-          files[fileName] = {
+          var dataChunk;
+          if (file.chunk.length == 1) {
+            dataChunk = file.chunk[0].data;
+          } else if (file.chunk.length > 1) {
+            // Does this ever happen?
+            dataChunk = "";
+          } else {
+            dataChunk = "";
+          }
```

Comment:
I don't think this part is required. Let's just leave data as `data: file.chunk[0].data` for now

Reply:
This caused the error for me.
_I had an empty file, which was saved by sacred but no chunk in fs.chunks was created._
Obviously, the else if is not necessary right now, but I felt like it's clearer. But I can remove the 'else if' if you want me to